### ISSUE TITLE
Side by Side Image comparison link does not work for svgs

### ISSUE THREAD

**Issue by delvh** [go-gitea/gitea#17112]
### Gitea Version

1.15.2

### Operating System

Linux

### Browser Version

Firefox 92.0

### Can you reproduce the bug on the Gitea demo site?

 https://try.gitea.io/delvh/kanban-test/pulls/21/files

### Description

The link to use side by side comparison of an existing with a changed image does not work anymore. The only way to see a side-by-side comparison is to reload the PR as then side-by-side will automatically be selected.
The cursor still appears as if you could click it.
However, when you actually click on it, nothing happens.
There is also nothing logged inside the console.

### Screenshots

Not possible in this case as that would be the same as requiring proof that a package did in fact not arrive.

**Comment by saitho** [comment]
@delvh Can you provide an example or reconstruct the issue on try.gitea.io or describe how to reproduce it?

**Comment by delvh** [comment]
Surprisingly, I cannot anymore.
Neither on try.gitea.io nor on a private Gitea server.
This leaves me with only three possible conclusions:
a) I hallucinated this malfunction. Unlikely.
b) It was fixed in the meantime, as now the private server is on 1.15.3. Also unlikely, because I wouldn't know which commit might have fixed it.
c) There is a weird edge case somewhere in the JS that occurs on 1 out of 1000 images, i.e. an improperly handled error. The likeliest option. Possibly even beyond the scope of Gitea, if an external library is used to deliver this functionality.
